### PR TITLE
[FR] Rename ReadingOrder to FormReadingOrder

### LINKED ISSUE #20787: Add FormRecognizer to ReadingOrder
Make the type unique

### PR DESCRIPTION
**Pull request by maririos**
Fixes: https://github.com/Azure/azure-sdk-for-net/issues/20787

### CODE REVIEW

**Review by kristapratico — APPROVED** [review]
Changelog? :)

**Comment by maririos** [comment]
>Changelog? :)

Thank you! Added

File: sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 ### Breaking changes
 - Renamed `Id` for `Identity` in all the `StartRecognizeIdDocuments` functionalities. For example, the name of the method is now `StartRecognizeIdentityDocuments`.
+- Rename  the model `ReadingOrder` to `FormReadingOrder`.
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md:10]: Should it be "Renamed" like the entry above? Also, nit: double space after "Rename".

**Review by kinelski — APPROVED** [review]
LGTM.

File: sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md (hunk 1 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md]: fixed

**Pull request merged**